addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -112,6 +112,9 @@ function factory(dependencies) {
             notifyServer = true,
             replaceNewMessage = false
         } = {}) {
+            if (!makeActive && this.env.messaging.device.isMobile) {
+                return;
+            }
```

Comment:
This will work, but according to what we said with Alex, we should remove this part, and not call the method instead when in that situation.

Reply:
Indeed, don't call `openThread` in the 1st place if it shouldn't open thread.